nemo_text_processing/text_normalization/normalize.py

```diff
@@ -455,17 +459,24 @@ def _process_batch(
 
             with open(f"{dir_name}/{batch_idx:06}.json", "w") as f_out:
                 for line in normalized_lines:
+                    if isinstance(line['normalized'], set):
+                        if len(line['normalized']) == 1:
```

Comment:
there's a variable `output_field` - please use it instead of "normalized"

Reply:
done